rest/rest-server/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/RMRestInterface.java

```diff
@@ -639,19 +687,28 @@ void setMBeanInfo(@HeaderParam("sessionid") String sessionId, @PathParam("name")
      * without redundancy, in a friendly JSON format.
      *
      * The following sources will be queried from the RRD DB :<pre>
-     * 	{ "BusyNodesCount",
-     *    "FreeNodesCount",
-     *    "DownNodesCount",
-     *    "AvailableNodesCount",
-     *    "AverageActivity" }</pre>
+     * 	{ AvailableNodesCount",
+     * 	"FreeNodesCount",
+     * 	"NeededNodesCount",
+     * 	"BusyNodesCount",
+     * 	"DeployingNodesCount",
+     * 	"ConfigNodesCount",
+     * 	"DownNodesCount",
+     * 	"LostNodesCount",
+     * 	"AverageActivity" };
+     * 	</pre>
      *
      *
      * @param sessionId a valid session
-     * @param range a String of 5 chars, one for each stat history source, indicating the time range to fetch
+     * @param range a String of 9 chars, one for each stat history source, indicating the time range to fetch
```

Comment:
10 Chars ?

Reply:
It is  9 which is number of the different datasource (AvailableNodesCount, FreeNodesCount.. ).

Reply:
Ok